Mail/MailApp.swift

```diff
@@ -107,7 +56,6 @@ struct MailApp: App {
                 if let composeMessageIntent {
                     ComposeMessageIntentView(composeMessageIntent: composeMessageIntent)
```

Comment:
Yes, it is.
At first I wanted to do it but it doesn't feel right to have the MainViewState when we are in a window. 
The ComposeMessageView works without MainViewState in the ShareExtension for example.

I think that we should / could refactor the ComposeMessageView to not need it at all but this would be in an other PR